tests/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_5_structural_analysis/test_formula_5_1.py

```diff
@@ -0,0 +1,54 @@
+"""Testing formula 5.1 of NEN-EN 1992-1-1+C2:2011."""
+# pylint: disable=arguments-differ
+import pytest
+
+from blueprints.codes.eurocode.nen_en_1992_1_1_c2_2011.chapter_5_structural_analysis.formula_5_1 import Form5Dot1Imperfections
+
```

Comment:
I miss testcases (integration tests) where the subformulas are inputted in the main formula

Reply:
Good one! I will add those.